eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php

```diff
@@ -128,23 +128,28 @@ protected function configure()
                 'since',
                 null,
                 InputOption::VALUE_OPTIONAL,
-                'Refresh changes since a time provided in any format understood by DateTime. Implies "no-purge", cannot be combined with "content-ids" or "subtree"'
+                'Refresh changes since a time provided in any format understood by DateTime. Implies "no-purge", cannot be combined with "content-ids", "subtree" or "content-type"'
```

Comment:
Punctuation: use Oxford comma here and in the other related places please
```suggestion
                'Refresh changes since a time provided in any format understood by DateTime. Implies "no-purge", cannot be combined with "content-ids", "subtree", or "content-type"'
```